README.md

```diff
@@ -16,6 +16,7 @@ This module is meant for use with Terraform 0.12. If you haven't
 [upgraded][terraform-0.12-upgrade] and need a Terraform
 0.11.x-compatible version of this module, the last released version
 intended for Terraform 0.11.x is [2.4.1].
+Modele isn't supported on Windows and/or outside of a bash environment.
```

Comment:
oops, typo.  Should be 'module'

These modules require a bash shell with python installed, but it probably otherwise does run in windows.  I tried it a few weeks back and basic functionality worked using the bash shell that gets installed with git by default.  However, I didn't exercise all of the functionality (gsuite integration) and/or run the integration tests, and I no longer have that windows machine to test with, so I can't confirm that all of the modules work correctly in windows when the dependencies are available.  

Terraform is kind of a pain in windows, regardless of the use of this module.  There are a lot of assumptions that get made about unix tools being available. You might find running it in a linux container works better for a wide variety of terraform modules, though mounting the host filesystem inside a container in docker for windows brings its own set of headaches, in my experience.